cluster-autoscaler/go.mod

```diff
@@ -342,35 +365,10 @@ replace github.com/mesosphere/konvoy/clientapis => github.com/mesosphere/konvoy/
 
 replace github.com/google/go-querystring => github.com/google/go-querystring v1.0.0
 
-replace github.com/mesosphere/kommander-cluster-lifecycle/clientapis => github.com/mesosphere/kommander-cluster-lifecycle/clientapis v0.0.0-20200310195319-9ade9d29b183
+replace github.com/mesosphere/kommander-cluster-lifecycle/clientapis => github.com/mesosphere/kommander-cluster-lifecycle/clientapis v0.0.0-20200323150606-ab6f57ecf9a3
 
 replace k8s.io/helm => k8s.io/helm v2.16.3+incompatible
 
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.0.0+incompatible
 
-replace (
-	k8s.io/api => k8s.io/api v0.16.4
```

Comment:
Why do we need to revert this?

Reply:
let me double check it, I just run the hack/update-vendor. Shouldn't we do that ?

Reply:
do we need go.mod-extra ?

Reply:
With https://github.com/mesosphere/autoscaler/commit/89add3fd526f454e71aa57f1f55a9d7fd6a7d320 we don't need to run hack/update-vendor.sh anymore.